src/components/Timeout/Timeout.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect, useRef } from 'react';
+import './timeout.css';
+
+export default function Timeout(props) {
+  //track idle time in background
```

Comment:
i'd recommend having one value `dtUpdated` who's value is a time
then count the difference between `Date.now()` and `dtUpdated`. if the difference is 1, then the user is idle, if the difference is 6, then we auto log them out. 

even though it's more math, this is seen more commonly due to db's storing Date types for stuff like this. this isn't tied to a db, but it'll be good practice

you can also use native javascript to get the minutes and seconds from milliseconds. 


check this out in your browser console:
```
const date = new Date()
date.getMinutes()
date.getSeconds()
```

or if you want, dayjs is really good for time related things https://day.js.org/docs/en/get-set/minute#docsNav